### PR TITLE
Add share icon from the design system

### PR DESCRIPTION
**Pull request by gurooerbech**
Adds the share icon from the design system to SATS DNA Icons.

Screenshot 2023-12-13 at 12 16 46

### CODE REVIEW

File: sats-dna/src/main/res/drawable/ic_share.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:pathData="M19,1.75C17.757,1.75 16.75,2.757 16.75,4C16.75,5.243 17.757,6.25 19,6.25C20.243,6.25 21.25,5.243 21.25,4C21.25,2.757 20.243,1.75 19,1.75ZM15.25,4C15.25,1.929 16.929,0.25 19,0.25C21.071,0.25 22.75,1.929 22.75,4C22.75,6.071 21.071,7.75 19,7.75C17.881,7.75 16.876,7.26 16.189,6.482L8.56,10.817C8.683,11.189 8.75,11.587 8.75,12C8.75,12.413 8.683,12.811 8.56,13.183L16.189,17.518C16.876,16.74 17.881,16.25 19,16.25C21.071,16.25 22.75,17.929 22.75,20C22.75,22.071 21.071,23.75 19,23.75C16.929,23.75 15.25,22.071 15.25,20C15.25,19.587 15.317,19.189 15.44,18.817L7.811,14.482C7.124,15.26 6.119,15.75 5,15.75C2.929,15.75 1.25,14.071 1.25,12C1.25,9.929 2.929,8.25 5,8.25C6.119,8.25 7.124,8.74 7.811,9.518L15.44,5.183C15.317,4.811 15.25,4.413 15.25,4ZM5,9.75C3.757,9.75 2.75,10.757 2.75,12C2.75,13.243 3.757,14.25 5,14.25C6.243,14.25 7.25,13.243 7.25,12C7.25,10.757 6.243,9.75 5,9.75ZM19,17.75C17.757,17.75 16.75,18.757 16.75,20C16.75,21.243 17.757,22.25 19,22.25C20.243,22.25 21.25,21.243 21.25,20C21.25,18.757 20.243,17.75 19,17.75Z"
+      android:fillColor="#14233C"
```
Comment (sindrenm) [inline sats-dna/src/main/res/drawable/ic_share.xml:8]: Let's default icon colors to black, and then tint at use-site. :sunglasses:

Comment (sindrenm) [reply sats-dna/src/main/res/drawable/ic_share.xml:8]:
```suggestion
      android:fillColor="#000000"
```

**Review by sindrenm — APPROVED** [review]
:tada:

**Review by cecilieopperudsats — APPROVED** [review]
👍

**Pull request merged**